y-rb.gemspec

```diff
@@ -37,11 +37,10 @@ Gem::Specification.new do |spec|
   spec.metadata["rubygems_mfa_required"] = "true"
 
   spec.add_dependency "rake", "~> 13.2"
-  spec.add_dependency "rb_sys", "~> 0.9.86"
+  spec.add_dependency "rb_sys", "~> 0.9.110"
 
   spec.add_development_dependency "base64"
   spec.add_development_dependency "rake-compiler", "~> 1.2.1"
-  spec.add_development_dependency "rake-compiler-dock", "~> 1.9.1"
```

Comment:
`rb_sys` now depends on a specific version of `rake-compiler-dock, so let's just use that as the hard dependnecy.